packages/core/test/logger.js

```diff
@@ -72,9 +72,9 @@ describe('logger', () => {
         token: options.token,
         message: 'test',
         data: {
-          password: ':censored:6:a5023f748d:',
```

Comment:
The tests are all the same, but I changed the default salt for the `censor` function, so the hashes had to be updated.